src/components/MyAvatarMenu/MyAvatarMenu.tsx

```diff
@@ -44,9 +44,18 @@ export const MyAvatarMenu = () => {
       <Hidden smDown>
         <Popover open={mouseEnterPopover}>
           <PopoverTrigger
+            tabIndex={-1}
             css={{ outline: 'none' }}
-            asChild
             ref={triggerRef}
+            onKeyDown={e => {
+              if (e.key === 'Enter') {
```

Comment:
This condition is different than the one in OverviewMenu. maybe a lil helper function that is like:
```
const onEnter = (f: func(e: uhhSomething): void) => {
 if (e.key === 'Enter' || e.key === ' ') {
   f(e);
 }
}
```

Reply:
good idea.  will add.